lib/fastlane/plugin/wpmreleasetoolkit/models/file_reference.rb

```diff
@@ -10,10 +10,7 @@ def initialize(params = {})
       end
 
       def source_contents
-        # TODO: This works only on CircleCI. I chose this variable because it's the one checked by Fastlane::is_ci.
-        # Fastlane::is_ci doesn't work here, so reimplementing the code has been necessary.
-        # (This should be updated if we change CI or if fastlane is updated.)
-        return File.read(secrets_repository_file_path) unless self.encrypt || ENV.key?('CIRCLECI')
+        return File.read(secrets_repository_file_path) unless self.encrypt || FastlaneCore::Helper.is_ci?
```

Comment:
Any idea why the comment initially said that "`Fastlane::is_ci?` doesn't work here "?

Reply:
I'm pretty sure the reason is that `is_ci` is [a Fastlane action](https://github.com/fastlane/fastlane/blob/e874a47c6e2e0e61590a03d3b71e75e5a505d1ce/fastlane/spec/actions_specs/is_ci_spec.rb) and can't be called directly.

If we put it in here, we get a no method error.

Reply:
Ah, right. As opposed to calling the helper directly as you did here 👍👌